make/photon/prepare/utils/chart.py

```diff
@@ -12,13 +12,12 @@
 def prepare_chartmuseum(config_dict):
 
     core_secret = config_dict['core_secret']
-    registry_custom_ca_bundle_path = config_dict['registry_custom_ca_bundle_path']
```

Comment:
How is this change relevant to this PR?  Is it necessary to remove it?

Reply:
because it is an unused var